src/en/components/checkbox/design.md

```diff
@@ -18,7 +18,7 @@ date: "git Last Modified"
 
 **The checkbox hint** gives users additional information about the choice.
 
-<img class="b-sm b-gray p-400" src="/images/en/anatomy/gcds-checkbox-anatomy.svg" alt=""/>
+<img class="b-sm b-gray p-400" src="/images/en/anatomy/gcds-checkbox-anatomy.svg" alt="Checkbox anatomy showing these labels Check box legend Text area label, Fieldset Hint, Checkmark, Checkbox Label, Checkbox Button and Checkbox Hint."/>
```

Comment:
Should this be: "the labels" instead of "these labels"?
Maybe "Checkbox legend," instead of "Check box legend" aka checkbox as one word since that is how we spell it everywhere else and add a comma after legend.